website_crm_privacy_policy/templates/contactus.xml

```diff
@@ -6,18 +6,18 @@
         name="Add privacy policy checkbox"
     >
         <xpath
-            expr="//form//div/div/span[@id='o_website_form_result']/../.."
+            expr="//form//div/div/span[@id='s_website_form_result']/.."
             position="before"
         >
-            <div class="form-group row form-field o_website_form_required_custom">
-                <div class="offset-lg-3 offset-md-4 col-md-8 col-lg-7">
+            <div class="form-group row form-field s_website_form_required_custom">
+                <div class="col-sm" style="margin-left: 200px;">
```

Comment:
Shouldn't the different `col-(sm|md|lg)-X` attributes be kept to ensure responsiveness? Could you give more insight on why you remove this?

Reply:
I don't really like this change.
Now, in v14 without making this change it would look like this:
![old](https://user-images.githubusercontent.com/4117568/141324563-a8c587e1-449a-4ccd-b7df-9db40f52de90.png)


In v13 the odoo code
https://github.com/odoo/odoo/blob/21eb4d1afdd0e45f3d9081a2ca7317a6eabeb392/addons/website_crm/views/website_crm_templates.xml#L44-L47
allowed to make that change.

Now in v14 the code is the following:
https://github.com/odoo/odoo/blob/d43b9923291dd103a80f552a38c395229a84d8db/addons/website_crm/views/website_crm_templates.xml#L83-L87

Any suggestions to improve this?

Reply:
Now it looks much better with @Tardo's suggestion. Thanks :+1: